Source/Bot/Bot+Blobs.swift

```diff
@@ -13,7 +13,7 @@ extension Bot {
     func blobsAndDatas(for blobs: Blobs,
                        completion: @escaping (([(Blob, Data)]) -> Void))
     {
-        Thread.assertIsMainThread()
+        //Thread.assertIsMainThread()
```

Comment:
I'm fine with removing these assertions (we can delete them, not only comment them) because we want to be able to call them from any thread (in fact, I already delete the sync and refresh ones because I need to call them from the background). I already discussed this with Henry, so we are fine. The only thing I want to tell you is that these should not be doing any performance harm, they just check we are calling the func from the main thread, I cannot believe the app is faster without these assertions, but still, it doesn't mind xd